<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>127</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>912</width>
    <height>127</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>912</width>
    <height>127</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="pgBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>871</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_go">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Go</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
